datashader/glyphs/line.py

```diff
@@ -61,6 +61,8 @@ class _AntiAliasedLine(object):
 
     def set_line_width(self, line_width):
         self._line_width = line_width
+        if hasattr(self, "antialiased"):
```

Comment:
This isn't excellent, but is the simplest solution available given that the `Glyph` class has the `antialiased` attribute and this `_AntiAliasedLine` class is a mixin for line glyphs only. In fact the `antialiased` flag is always available here given how we are using it, so this `hasattr` isn't really needed but it is good practice to check before setting it.

In the long run, when we add antialiasing to other glyphs such as points it would be better to remove the `_AntialisedLine` class and move the `line_width` to the `Glyph` base class. But it will need a more generic name than `line_width` (`antialiased_spread`?) which is TBD.